README.md

```diff
@@ -1,87 +1,71 @@
-val recyclerView: RecyclerView? by bind(R.id.recycler)
+2. For using HMOSUtils in separate application using har file, add the har file in the entry/libs folder and add the dependencies in entry/build.gradle file.
+```groovy
+	dependencies {
+        implementation fileTree(dir: 'libs', include: ['*.jar'])
+        testImplementation 'junit:junit:4.13'
+    }
 ```
 
-Also: `View.setBackgroundTint(color)` will wrap & tint a view's background drawable. 
-
-#### Prefs
-
-A bunch of generic-typed preference utility functions.
-
-```kotlin
-// editing a SharedPreferences object (automatically runs .apply())
-sharedPrefs.edit {
-    putInt("themeKey", THEME_DARK)
-}
+## Usage
 
-// setter/getter operator functions
-val theme: Int = sharedPrefs["themeKey"]
-sharedPrefs["themeKey"] = THEME_LIGHT
+#### Include code in your layout:
+
+```xml
+<Button
+        ohos:height="match_content"
+        ohos:width="match_content"
+        ohos:id="$+id:buttonimageutil"
+        ohos:below="$id:buttondimen"
+        ohos:right_of="$id:button"
+        ohos:top_margin="30vp"
+        ohos:top_padding="15vp"
+        ohos:bottom_padding="15vp"
+        ohos:left_padding="38vp"
+        ohos:right_padding="38vp"
+        ohos:left_margin="20vp"
+        ohos:text="Image Utils"
+        ohos:text_size="28vp"
+        ohos:text_color="#FFF8F4F4"
+        ohos:background_element="#FF03CACA"
+        />
 ```
 
-Property delegates can be used to greatly simplify SharedPreferences operations.
+## DimenUtils
 
-```kotlin
-// binding a single property
-val theme by sharedPrefs.get<Int>("themeKey", defaultValue = THEME_LIGHT)
+includes some unit conversion functions that use getDeviceCapability() or getDefaultDisplay() to find the display density:
 
-// uses the delegated property name ("themeKey") if a key is not provided
-val themeKey by sharedPrefs.get<Int>(defaultValue = THEME_LIGHT)
-```
-
-##### TypedPreference
+1. dpToPx(Float) : Int
 
-TypedPreference objects can be used to create a set of typed "pointers" to reference throughout your project, instead of relying on a hardcoded preference key/string.
+2. spToPx(Float) : Int
 
-```kotlin
-// constructs a TypedPreference<Int> with the key "PREF_THEME"
-val PREF_THEME by pref<Int>(defaultValue = THEME_LIGHT)
+3. pxToDp(Int) : Float
 
-// in another file...
-var theme by sharedPrefs.get(PREF_THEME)
-
-// alternatively, in an activity/fragment/etc (if using PreferenceManager.getDefaultSharedPreferences)
-var theme by prefs(PREF_THEME)
-```
+4. pxToSp(Int) : Float
 
-#### DimenUtils
 
-Contains two `Context` extension functions: `getStatusBarHeight()` and `getNavigationBarHeight()`, that both work accordingly.
+## Future Work
 
-Also includes some unit conversion functions that use `Resources.getSystem()` to find the display density:
-- `dpToPx(Float) : Int`
-- `spToPx(Float) : Int`
-- `pxToDp(Int) : Float`
-- `pxToSp(Int) : Float` 
+1. Seekbar is not implemented as some of the android mapping were not found 
```

Comment:
Clearly Mention the missing apis in HMOS platform.

Reply:
Done